platform-sdk/consensus-otter-tests/build.gradle.kts

```
@@ -46,8 +47,10 @@ tasks.register<Test>("testTurtle") {
     jvmArgs("-XX:ActiveProcessorCount=6")
 }
 
-// Runs tests against the Solo environment
-tasks.register<Test>("testSolo") {
+// Runs tests against the Container environment
+tasks.register<Test>("testContainer") {
+    dependsOn(":consensus-otter-docker-app:assemble")
+
     useJUnitPlatform {}
```

```suggestion
    useJUnitPlatform()
    testClassesDirs = sourceSets.test.get().output.classesDirs
    classpath = configurations.testRuntimeClasspath.get()
```

Be explicit on `testClassesDirs` and `classpath` for newly registered Test tasks. Without this, you see a deprectaion and this will break soon when we move to Gradle 9.